mock-api/msw/handlers.ts

```diff
@@ -486,6 +486,33 @@ export const handlers = makeHandlers({
       time_run_state_updated: new Date().toISOString(),
     }
 
+    body.external_ips?.forEach((ip) => {
+      if (ip.type === 'floating') {
+        const floatingIp = lookup.floatingIp({
+          project: project.id,
+          floatingIp: ip.floating_ip,
+        })
+        if (floatingIp.instance_id) {
+          throw 'floating IP cannot be attached to one instance while still attached to another'
+        }
+        floatingIp.instance_id = instanceId
+      } else if (ip.type === 'ephemeral') {
+        const poolId = db.ipPools.find((pool) => pool.name === ip.pool)?.id
```

Comment:
use `lookup.ipPool`. the result is non-nullable because it throws the 404 if it doesn't find it